tests/research/test_b2_01_volatility_transition.py

```
@@ -0,0 +1,275 @@
+        i=6,
+        replicate=7,
+        t_ms=t_ms,
+        target=target,
+        level_state=level,
+        transition_state=transition,
+        w_min=60,
+        d_min=60,
+        h_min=30,  # h_steps=2, so source j must be <=4.
+        ref_steps=6,
+        candidate_min_count=1,
+    )
```

Ruff (0.16.2)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)